src/shogun2-core/src/main/java/de/terrestris/shogun2/service/GeoServerInterceptorService.java

```diff
@@ -320,8 +320,8 @@ public boolean evaluate(InterceptorRule rule) {
 					"least the basic sets of rules (e.g. ALLOW all WMS " +
 					"requests) when using the interceptor.");
 			throw new InterceptorException("No interceptor rule found.");
-		} else {
-			LOG.debug("Identified the following rule as most the specific " +
+		} else if(LOG.isTraceEnabled()) {
+			LOG.trace("Identified the following rule as most the specific " +
 					"one: " + mostSpecificRule);
```

Comment:
I thi the toString method of the rules will result in added newlines here. while we are at it, I'd like to have only one line of logging here.

This is non blocking of course.

Reply:
Yes. Maybe newlines would be added here, but this depends on the `toString` implementation of the `InterceptorRule` model, right?

The if-condition to check whether trace is enabled has been added to avoid that some kind of lazy loading (of the rule) would be triggered, but i think that this will not happen as the `InterceptorRule` model does not have any dependent sub-entities (but only primitive fields).